https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=1092883

Description: This was the minimal patch I used to untangle python3.13/libzim/kiwix transition.

--- a/src/server/kiwix-serve.cpp
+++ b/src/server/kiwix-serve.cpp
@@ -413,7 +413,7 @@
     exit(1);
   }
 
-  std::string url = "http://" + server.getAddress() + ":" + std::to_string(server.getPort()) + normalizeRootUrl(rootLocation);
+  std::string url = "http://" + server.getAddress().addr + ":" + std::to_string(server.getPort()) + normalizeRootUrl(rootLocation);
   std::cout << "The Kiwix server is running and can be accessed in the local network at: "
             << url << std::endl;
 
--- a/meson.build
+++ b/meson.build
@@ -18,8 +18,8 @@
 thread_dep = dependency('threads')
 libzim_dep = dependency('libzim', version:'>=9.0.0', static:static_linkage)
 libzim_dep = dependency('libzim', version:'<10.0.0', static:static_linkage)
-kiwixlib_dep = dependency('kiwix', version:'>=13.0.0', static:static_linkage)
-kiwixlib_dep = dependency('kiwix', version:'<14.0.0', static:static_linkage)
+kiwixlib_dep = dependency('libkiwix', version:'>=14.0.0', static:static_linkage)
+kiwixlib_dep = dependency('libkiwix', version:'<15.0.0', static:static_linkage)
 
 all_deps = [thread_dep, kiwixlib_dep, libzim_dep]
 
